plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -345,4 +354,9 @@ public static boolean isProjectionPushdownEnabled(ConnectorSession session)
     {
         return session.getProperty(PROJECTION_PUSHDOWN_ENABLED, Boolean.class);
     }
+
+    public static Optional<Long> getTargetMaxFileSize(ConnectorSession session)
```

Comment:
i asked you to have `Optional` here, but actually ... -- https://github.com/trinodb/trino/pull/10978